src/port1.0/portconfigure.tcl

```diff
@@ -404,12 +404,14 @@ proc portconfigure::configure_start {args} {
 # configure.universal_archs based on supported_archs and build_arch or
 # universal_archs, plus the SDK being used
 proc portconfigure::choose_supported_archs {archs} {
-    global supported_archs configure.sdk_version
+    global supported_archs configure.sdk_version macosx_version
 
     if {${configure.sdk_version} ne ""} {
         # Figure out which archs are supported by the SDK
         if {[vercmp ${configure.sdk_version} 11.0] >= 0} {
             set sdk_archs [list arm64 x86_64]
+        } elseif {[vercmp ${configure.sdk_version} 10.14] >= 0 && $macosx_version eq "10.14" && [file exists /usr/include/sys/cdefs.h]} {
```

Comment:
This still isn't quite correct; instead of `[vercmp ${configure.sdk_version} 10.14] >= 0` it should be `${configure.sdk_version} eq "10.14"`.

Reply:
done